resources/assets/js/models/types.ts

```diff
@@ -353,6 +358,16 @@ export type Experience =
   | (ExperienceAward & { type: "award" })
   | (ExperiencePersonal & { type: "personal" });
 
+export interface ExperienceSkill {
+  skill_id: number;
+  experience_id: number;
+  experience_type: string;
+  experience: Experience | null;
```

Comment:
I did it in this case mostly because there isn't an easy reference between an ExperienceSkill and an Experience... but also because it was easy 😆 

I'm not too concerned about providing the info in the API response or normalizing and querying for the related Experience on the frontend, so whichever way you're leaning is fine.

Reply:
Yeah, I think we should remove the nested Experience. We already pass in a list of Skills and use helper functions to link them to ExperienceSkill objects, we can do the same with Experience and ExperienceSkill. It'll make it easier to manage in the store, I think.

Reply:
The only downside with looking up ExperienceSkill is the way they're stored currently... you need to do two lookups in order to grab the right Experience (the type and the ID)